docs/_newsfragments/2355.newandimproved.rst

```diff
@@ -1,10 +1,14 @@
-Added a new flag :attr:`~falcon.ResponseOptions.xml_error_serialization` that
-can be used to disable XML serialization of :class:`~falcon.HTTPError` when
-using the default error serializer (and the client preferred it).
-This flag currently defaults to ``True``, keeping the same behavior as the previous
-Falcon series; Falcon 5.0 will change the default to ``False``, with the objective of
-completely removing support for it in a future version.
-User that whish to retain support for XML serialization of the errors in the default
-error serializer should add an XML media handler.
+A new flag, :attr:`~falcon.ResponseOptions.xml_error_serialization`, has been
+added to :attr:`~falcon.ResponseOptions` that can be used to disable automatic
+XML serialization of :class:`~falcon.HTTPError` when using the default error
+serializer (and the client prefers it).
 
-Deprecated the :meth:`~falcon.HTTPError.to_xml` method.
+This new flag currently defaults to ``True``, preserving the same behavior as
+the previous Falcon versions. Falcon 5.0 will either change the default to
+``False``, or remove the automatic XML error serialization altogether.
```

Comment:
you may want to sync also the xml_error_serialization doc-string to report either option for falcon 5